client/src/sass/include/_variables.scss

```diff
@@ -127,9 +127,9 @@ $zindex: (
   miniature    :    10,
   privacymsg   :    20,
   typeahead    :    30,
-  header       :  1000,
-  menu         : 11000,
   dropdown     : 12000,
+  header       : 12500,
+  menu         : 12600,
   popover      : 13000,
```

Comment:
popover, tooltip and help-popover should also be smaller than header and menu ?

Reply:
Also for z-index search instructions :
![Capture d’écran du 2020-08-20 23-38-40](https://user-images.githubusercontent.com/1877318/90829133-7c88b400-e33f-11ea-9476-73ec5c974f59.png)

z-index value for menu should be smaller than header and the content overlay in https://github.com/Chocobozzz/PeerTube/blob/hotfix/z-index/client/src/sass/bootstrap.scss#L179 should have `z-index: z('menu') - 1;` (instead of `z('header') -1`)